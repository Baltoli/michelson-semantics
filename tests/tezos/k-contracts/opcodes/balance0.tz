$contract { parameter unit;
storage mutez;
code {DROP; BALANCE; NIL operation; PAIR};
 } ;
$param Unit ;
$storage 111 ;
$ExpectedOutput 4000000000000 ;

