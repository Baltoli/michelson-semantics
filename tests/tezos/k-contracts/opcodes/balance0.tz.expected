<generatedTop>
  <k>
    #VerifyStorage ( #Mutez ( 4000000000000 ) )
  </k>
  <stack>
    Pair .List #Mutez ( 0 )
  </stack>
  <paramtype>
    unit .AnnotationList
  </paramtype>
  <storagetype>
    mutez .AnnotationList
  </storagetype>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( "InvalidMyAddr" )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( "InvalidSourceAddr" )
  </sourceaddr>
  <senderaddr>
    #Address ( "InvalidSenderAddr" )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
